### DISCUSSION TITLE
Setting up Kratos

### DISCUSSION THREAD

**Discussion by philipgierszal** [ory/kratos#2416] [Ideas]
Hi everybody,
I am trying to set up Kratos through docker, i found a guide [here ](https://www.ory.sh/docs/kratos/guides/docker#image-variants), unfortunately when trying to migrate my mysql DB, after executing the command `docker -e DSN="<your database URL>" run oryd/kratos:<version> migrate sql -e` this error pops up:
`unknown shorthand flag: 'e' in -e
See 'docker --help'.`

**Comment by gen1us2k** [comment]
Hello. We have a quickstart guide for docker. You can find [it here](https://www.ory.sh/docs/kratos/quickstart)
